### DISCUSSION TITLE
你们部署显示的是什么模型啊？好像不是GeminiPro啊

### DISCUSSION THREAD

**Discussion by pengshp** [babaohuang/GeminiProChat#48] [Q&A] [answered]
我部署了，问它使用的是什么大语言模型，好像不对啊
![图片](https://github.com/babaohuang/GeminiProChat/assets/13529131/7f0e3f26-aa4a-470e-b866-1f6e8064bd1e)

让它介绍自己也说自己是 `Bard` ,是我打开的方式不对吗？
![图片](https://github.com/babaohuang/GeminiProChat/assets/13529131/ee050282-1900-49fc-8b60-cedb69516382)

**Comment by babaohuang** [comment, marked as answer]
这是常见的大语言模型的幻觉，其实，它回答自己是什么很重要吗？

**Reply by laorent** [reply]
大佬，我的WiFi环境没问题，流量打开页面就出现User location is not supported for the API use，请问是什么回事，我需要重新部署吗，还是需要等代更新

**Reply by pengshp** [reply]
好像需要美国的IP地址才能用，你用流量就是国内的IP了，所以不支持

**Reply by babaohuang** [reply]
User location is not supported for the API use在文档中有说明了解决方案

**Comment by liceal** [comment]
![image](https://github.com/babaohuang/GeminiProChat/assets/33884851/466eacfb-623c-4bc9-b997-d97780f98c47)

**Reply by pengshp** [reply]
这个模型真是奇葩啊，哈哈😃

**Reply by laorent** [reply]
我发现也是，但是一刷新就变成谷歌模型了！！！！

**Comment by zxjack** [comment]
在 bard 上是可以知道今天的日期，这个模型不行，而且不能上网

**Reply by leealu** [reply]
![image](https://github.com/babaohuang/GeminiProChat/assets/103651858/1d766fb8-917f-4e10-9733-4bd07f3739c9)
